modules/_labs/badge/react/lib/Badge.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import {keyframes} from '@emotion/core';
+import styled from '@emotion/styled';
+import {borderRadius, colors, fontFamily} from '@workday/canvas-kit-react-core';
+
+export type BadgeVariant = 'default' | 'inverse';
+
+export interface BadgeProps extends React.HTMLAttributes<HTMLSpanElement> {
+  'aria-label': string;
+  count?: number;
+  variant?: BadgeVariant;
+}
+
+const variants = {
+  default: {
+    background: colors.cinnamon500,
+    color: colors.frenchVanilla100,
+    textShadow: '0 0 1px rgba(0,0,0, 0.35)',
+  },
+  inverse: {
+    background: colors.frenchVanilla100,
+    boxShadow: '0 1px 2px rgba(0,0,0, 0.25)',
+    color: colors.blueberry400,
+  },
+};
+
+const grow = keyframes`
+  from {
+    transform: scale(0.85);
+  }
+
+  to {
+    transform: scale(1.0);
+  }
+`;
+
+const Container = styled('span')<BadgeProps>(
+  {
+    alignItems: 'center',
+    animation: `${grow} 0.2s ease`,
+    borderRadius: borderRadius.circle,
+    boxSizing: 'border-box',
+    display: 'inline-flex',
+    fontFamily,
+    fontSize: '11px',
+    fontWeight: 700,
+    height: '20px',
+    justifyContent: 'center',
+    minWidth: '20px',
+    padding: '0 6.5px',
+  },
+  ({variant = 'default'}) => ({
+    ...variants[variant],
+  })
+);
+
+const Badge = (props: BadgeProps) => {
+  const {count = 0, ...elemProps} = props;
```

Comment:
you can add the variant default here

Reply:
I thought about that, but it gets used [here](https://github.com/Workday/canvas-kit/pull/566/files#diff-53b9d8f6bc67ea5a52fc795e9c4bb0bdR52), so I opted to pass it along with `{...elemProps}`. Unless you want it to be explicit?

Reply:
ah this isn't yet a common practice cause we just updated this, but @anicholls came up with a type that makes typescript happy so when you pass a default down below, it won't complain in the styled component. That just got merged into pre release v4 https://github.com/Workday/canvas-kit/blob/prerelease/v4/modules/common/react/lib/types.ts#L30

I think for now it this works! not a blocker for me

Reply:
Oh, that's nice! Thanks!